gamefixes-steam/517710.py

```diff
@@ -0,0 +1,8 @@
+"""Game fix Redout: Enhanced Edition"""
+
+from protonfixes import util
+
+
+def main() -> None:
+    # Requires vcrun2019 to launch
+    util.protontricks('vcrun2019')
```

Comment:
Source:

- https://steamdb.info/app/517710/
- https://www.protondb.com/app/517710?device=steamDeck

Reply:
I only have this title on Epic yes.
On Steamdb I saw the last update of the game was in 2022, and there was a report 1 year ago on Protondb that said vcrun2019. Maybe it was someone using the Epic and it wasn't mentioned.

Reply:
> On Steamdb, I saw the last update of the game was in 2022

While the game hasn't been updated, Proton / Wine has. So that's not really a good indicator.

> Maybe it was someone using the Epic and it wasn't mentioned.

Possible. Let's assume it works on Steam, as evidence implies.

Reply:
Furthermore, inspecting the depot for that title, one can see vcrun2019 as part of the game’s redistributables.

https://steamdb.info/app/517710/depots/

Reply:
Please revert the Steam fix and keep it local to the EGS version.